### PR TITLE
AIRNEIS-114 - New + Rework : Add styling features to linter

### PR DESCRIPTION
**Pull request by cedricgautier**
# DEPENDENCIES
- https://github.com/KrispyTech/airneis/pull/73
- https://github.com/KrispyTech/airneis/pull/75

# DONE 

- Added more styling linters in linter config
- Updated MakeFile

Refactored directories : 
- `lib/shared`
- `pkg/config`
- `src/controllers`
- `src/models`

Clean up - Styling : 
- `front/.env.local.exemple` : remove blankspace

### CODE REVIEW

**Review by avetisk — APPROVED** [review]
Noice 👌

File: Makefile [hunk 1]
```diff
@@ -81,7 +81,7 @@ restart-db:
 # You need to have bun installed to make it worked
 ci:
 	@echo "Running the same commands as the CI. Let's check the backend first"
-	@cd back && golangci-lint run && go build && rm airneis
+	@cd back &&  golangci-lint run --color always --sort-results && go build && rm airneis
```
Comment (avetisk) [inline Makefile]: Extra space.

Comment (cedricgautier) [reply Makefile]: Ah thanks!

**Review by BarnabePILLIAUDIN — APPROVED** [review]
LGTM! Brillant

File: back/lib/shared/helpers/strings.go
Comment (KyDev25) [inline back/lib/shared/helpers/strings.go:1]: What is this function used for?

Comment (cedricgautier) [reply back/lib/shared/helpers/strings.go:1]: This  command line is used to be able to run the CI locally

Comment (cedricgautier) [reply back/lib/shared/helpers/strings.go:1]: Which one ?

**Review by KyDev25 — COMMENTED** [review]
After the answer to my question, I will approve your review.

File: back/lib/shared/helpers/strings.go
Comment (KyDev25) [reply back/lib/shared/helpers/strings.go:1]: This one:

```go
func Capitalize(text string) string {
	caser := cases.Title(language.AmericanEnglish)

	return caser.String(text)
}
```

Comment (cedricgautier) [reply back/lib/shared/helpers/strings.go:1]: It's to capitalize a string

Comment (KyDev25) [reply back/lib/shared/helpers/strings.go:1]: Why "AmericanEnglish"?

Comment (cedricgautier) [reply back/lib/shared/helpers/strings.go:1]: Because we use American English as a standard in our code.

Comment (KyDev25) [reply back/lib/shared/helpers/strings.go:1]: Okay, and when we use another language, it will be dynamic?

Comment (cedricgautier) [reply back/lib/shared/helpers/strings.go:1]: This is not used for i18N, this function is only used for logs and such in our code base. So it does not need to be dynamic, we only use one language in our code base and it's American English

Comment (KyDev25) [reply back/lib/shared/helpers/strings.go:1]: I understand, thank you!

**Review by KyDev25 — APPROVED** [review]
Thank you for your answers, It's a very good job for me.

File: back/lib/shared/helpers/setStatusAndMessages.go [hunk 2]
```diff
@@ -0,0 +1,9 @@
+package helpers
+
+import "github.com/gofiber/fiber/v2"
+
+func SetStatusAndMessages(status int, message string, ctx *fiber.Ctx) error {
```
Comment (avetisk) [inline back/lib/shared/helpers/setStatusAndMessages.go:5]: When using generic words like `Status`, please refine them by adding what they are related to?

HTTP status? Order status?

Comment (cedricgautier) [reply back/lib/shared/helpers/setStatusAndMessages.go:5]: I'll make a rework to make it clearer. Thanks!

**Pull request merged**